e2e/testinfra/terraform/common/service_accounts.tf

```diff
@@ -78,3 +78,25 @@ resource "google_service_account_iam_member" "e2e_metric_writer_gke_binding" {
   role               = "roles/iam.workloadIdentityUser"
   member             = "serviceAccount:${data.google_project.project.project_id}.svc.id.goog[config-management-monitoring/default]"
 }
+
+resource "google_project_iam_member" "root-reconciler-wi-sa-iam" {
+  for_each = toset([
+    "roles/source.reader",
+    "roles/artifactregistry.reader",
+    "roles/storage.objectViewer",
+  ])
+  role    = each.value
+  member             = "serviceAccount:${data.google_project.project.project_id}.svc.id.goog[config-management-system/root-reconciler]"
+  project = data.google_project.project.id
+}
+
+resource "google_project_iam_member" "root-reconciler-fwi-sa-iam" {
+  for_each = toset([
+    "roles/source.reader",
+    "roles/artifactregistry.reader",
+    "roles/storage.objectViewer",
+  ])
+  role    = each.value
+  member             = "serviceAccount:cs-dev-hub.svc.id.goog[config-management-system/root-reconciler]"
```

Comment:
The `cs-dev-hub` IAM should probably be specific to the prow environment rather than in `common`. I don't think we can assume any dev environment will be able to set up IAM with this hard coded project.

Reply:
That means dev environments won't be able to run the e2e tests with Fleet WI.

Reply:
Yes, and that's the case already today. The dev environment is intended to be provisionable/usable by anyone, including an external contributor.

If we want to make fleet WI identity testable in a more generic sense, we would need to parametrize the fleet project in a number of places instead of hard coding `cs-dev-hub`. I think that is a bigger change that is out of scope here.

Reply:
Make sense and moved to the `prow` directory.